common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/wizards/NewProjectWizard.xtend

```diff
@@ -44,22 +52,38 @@ class NewProjectWizard extends BasicNewProjectResourceWizard {
 
 	override init(IWorkbench bench, IStructuredSelection selection) {
 		super.init(bench, selection)
-		windowTitle = "Create test-first Testproject"
+		windowTitle = "Create test-first project"
+		projectContentGenerator = new ProjectContentGenerator()
+	}
+
+	override addPages() {
+		super.addPages()
+		prjCfgPage = new TestProjectConfigurationWizardPage("projectcfg")
+		prjCfgPage.availableBuildSystems = projectContentGenerator.availableBuildSystems
+		prjCfgPage.availableFixtureNames = projectContentGenerator.availableFixtureNames
+		addPage(prjCfgPage)
 	}
 
 	override performFinish() {
 		val result = super.performFinish()
 
-		val srcFolder = newProject.createOrGetDeepFolder(SRC_FOLDER)
+		newProject.createOrGetDeepFolder(SRC_FOLDER)
 		newProject.addNature(JavaCore.NATURE_ID)
-		JavaCore.create(newProject) =>
-			[
-				val rawPath = PreferenceConstants.defaultJRELibrary +
-					#[JavaCore.newSourceEntry(srcFolder.fullPath),
-						JavaCore.newContainerEntry(JUnitCore.JUNIT4_CONTAINER_PATH)]
-				setRawClasspath(rawPath, null)
-			]
+		JavaCore.create(newProject)
 		newProject.addNature(XtextProjectHelper.NATURE_ID)
+
+		if (!prjCfgPage.selectedFixtures.isEmpty) {
+			new ProgressMonitorDialog(Display.current.activeShell).run(false, false,
+				new IRunnableWithProgress() {
+
+					override run(IProgressMonitor monitor) throws InvocationTargetException, InterruptedException {
```

Comment:
Those three lines above should be extracted to a helper method. We will probably reuse this code sometime and can test it this way.